src/rust/ensogl/lib/components/src/text_label.rs

```diff
@@ -0,0 +1,198 @@
+//! Text label component. Appears as text with background.
+
+use crate::prelude::*;
+
+use enso_frp as frp;
+use enso_frp;
+use ensogl_core::application::Application;
+use ensogl_core::data::color;
+use ensogl_core::display::shape::*;
+use ensogl_core::display::traits::*;
+use ensogl_core::display;
+use ensogl_text as text;
+
+
+// ==========================
+// === Shapes Definitions ===
+// ==========================
+
+
+// === Constants ===
+
+const TEXT_OFFSET : f32 = 10.0;
+const TEXT_SIZE   : f32 = 12.0;
+const HEIGHT      : f32 = TEXT_SIZE * 3.0;
+const PADDING     : f32 = 15.0;
+const SHADOW_SIZE : f32 = 10.0;
```

Comment:
please lets use that from our color theme. Also with other things that make sense, like text size, it would be cool as well.